internal/caller/linux/linuxconfig.go

```diff
@@ -0,0 +1,136 @@
+package caller
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
+)
+
+var (
+	installPath   = "/etc/wireguard"
+	tunnelTemplate = `[Interface]
+PrivateKey = %s
+Address = %s
+	
+[Peer]
+PublicKey = %s
+AllowedIPs %s
```

Comment:
Missing an equals sign.  This would also be better as a `text/template` since you could then use named variables in it rather than needing to know the order the strings go in.

Reply:
Since this string will be needed for at least two of the configurators, should it be moved up to configurator.go?